github_stargazers/github.py

```diff
@@ -33,9 +39,11 @@ class GitHub:
     __GITHUB_URL: str = "https://github.com"
     __STARGAZERS_URL_SUFFIX: str = "/stargazers"
     __PAGE_SUFFIX: str = "?page="
+    __MARK_END_OF_STARGAZERS: typing.List[str] = ['This repository has no more stargazers.']
+
     __OK_STATUS_CODE: int = 200
     __TOO_MANY_REQUESTS_STATUS_CODE: int = 429
-    __MARK_END_OF_STARGAZERS: typing.List[str] = ['This repository has no more stargazers.']
+    __NOT_FOUND: int = 404
```

Comment:
Let's take advantage of consistency with the rest of constants above about HTTP status codes by renaming it to `__NOT_FOUND_STATUS_CODE`.